packages/models-library/src/models_library/api_schemas_directorv2/clusters.py

```diff
@@ -72,7 +73,7 @@ class Scheduler(BaseModel):
     status: str = Field(..., description="The running status of the scheduler")
     workers: WorkersDict | None = Field(default_factory=dict)
 
-    @validator("workers", pre=True, always=True)
+    @field_validator("workers", mode="before")
```

Comment:
See: https://docs.pydantic.dev/latest/migration/#changes-to-validators

> If you use the `always=True` keyword argument to a validator function, note that standard validators for the annotated type will also be applied even to defaults, not just the custom validators. For example, despite the fact that the validator below will never error, the following code raises a `ValidationError`.

> To avoid this, you can use the validate_default argument in the Field function. When set to True, it mimics the behavior of `always=True` in Pydantic v1. However, the new way of using `validate_default` is encouraged as it provides more flexibility and control.

Reply:
so if I get right, validate_default means it will validate the default value on instance creation. I don't quite get whether it will validate everytime.